src/components/general/mediaTypes/mediaTypes.module.scss

```diff
@@ -1,3 +1,12 @@
 .organisationLogo {    
     filter: grayscale(100%);
+}
+
+.annotoriousPopUp {
+    width: 400px;
```

Comment:
Isn't fixed px width tricky, especially with eye on responsiveness?

Reply:
Yes in principle, this element is positioned absolute, which means it will hover over everything but in a certain position.
Setting a certain width is needed for absolute elements, percentages do work, but then it needs to be position relative to the parent, in this case the annotorious component.
In version 2 this pop-up was automatically appointed to a width of 400px, that is why I took over this value.
We can make it responsive, but I doubt it would be better for this specific case.

Reply:
Ok sure, let's work with this and change it if we hit issues